fideslog/sdk/python/event.py

```diff
@@ -123,3 +123,43 @@ def __init__(
 
         except AssertionError as err:
             raise InvalidEventError(str(err)) from None
+
+
+class UserRegistrationEvent:
+    """
+    A discrete event, representing a user action within a fides tool.
+    """
+
+    def __init__(
+        self,
+        email: str,
+        organization: str,
+        registered_at: datetime,
+    ) -> None:
+        """
+        Define a new user registration event to send to the fideslog server.
+
+        :param email: User email
+        :param organization: User organization
+        :param registered_at: The UTC timestamp when the event occurred, in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format. Must include the UTC timezone, and represent a datetime in the past.
+
+        """
+
+        try:
+            assert email is not None and email.find("@") != -1
+            self.email = email
+
+            assert organization is not None
```

Comment:
Same here - help your users out by providing an explanation for the failures if/when they occur.